src/server/ua_server_ns0_gds.c

```diff
@@ -15,6 +15,7 @@
 typedef struct UA_FileContext {
     LIST_ENTRY(UA_FileContext) listEntry;
     UA_ByteString file;
+    UA_ByteString dataToWrite;
```

Comment:
I think I know what this does.
Knowing is better.
Please a comment to explain the struct member.